webapp/src/js/enketo/translator.js

```diff
@@ -8,9 +8,7 @@ function translate(key) {
     return moment().month(parseInt(key.substring(11)) - 1).format('MMM');
   }
 
-  const angularServices = angular.element( document.body ).injector();
-  const $translate = angularServices.get( '$translate' );
-  return $translate.instant('enketo.' + key);
+  return window.CHTCore.Translate.instant('enketo.' + key);
```

Comment:
I don't think we should null-check, it should always be available. I guess we could check if translations are loaded _before_ loading the form, but this wasn't necessary before. I'll experiment a little and see. I don't think it's possible to refactor this to be async.

Reply:
I've had a look and I'm not sure if we should change behavior here. The function that we're returning here can't be async. The only option would be to wait until translations are loaded and only then render the form. 
This race also exists in the old codebase, so I'm tempted to leave it as is, and keep the forms loading even without translations to avoid generating other bugs.